controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -374,10 +376,12 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
     }
 
     //TODO autoconfig process the rest of the class
+    //TODO autoconfig add support for none intel agents
```

Comment:
The correct spelling is non-Intel, not "none".

Reply:
will fix in the squash